sample/src/main/java/oracle/r2dbc/samples/TcpsConnectDemo.java

```diff
@@ -114,7 +115,7 @@ public static void main(String[] args) throws URISyntaxException {
         .option(ConnectionFactoryOptions.DRIVER, "oracle")
         .option(ConnectionFactoryOptions.HOST, HOST)
         .option(ConnectionFactoryOptions.PORT, PORT)
-        .option(ConnectionFactoryOptions.DATABASE, DatabaseConfig.SERVICE_NAME)
```

Comment:
Just curious why only the service name comes from DatabaseConfig? Why not host, port, etc?

Reply:
Ah, the worst enemy of code reviewers strikes again: There are static imports for HOST, PORT, but not for SERVICE_NAME. I corrected this in the next commit, now SERVICE_NAME is a static import as well.